src/styles/page-template/index.md.njk

```diff
@@ -98,6 +98,13 @@ Variables can be set with:
       <td class="govuk-table__cell">Add a class to the <code>&lt;html&gt;</code> element.</td>
     </tr>
 
+    <tr class="govuk-table__row">
```

Comment:
Could we move these 2 new rows up so they sit directly underneath the `htmlLang` row? They seem to belong together in a group of 3, as they all relate to each other.

Reply:
@m-green Just a heads up that so far we've put these parameters in the order that they appear in the template file so the suggested change would diverge from that. To my knowledge (spoke to @nickcolley about this as well) there wasn't any particular reason to do so, but it's what we have right now. But happy to re-order these particular parameters if you think it makes sense.

Reply:
Ah ok - makes sense! 👍